airdrop/airdrop-distributor/tests/functions/claim.rs

```diff
@@ -235,17 +243,19 @@ mod revert {
     async fn panics_when_claim_twice_manual_tree() {
         let (deploy_wallet, wallet1, wallet2, wallet3, asset, claim_time) = setup().await;
 
-        let airdrop_leaves: [([u8; 32], u64); 3] = [
-            (*wallet1.wallet.address().hash(), 1),
-            (*wallet2.wallet.address().hash(), 2),
-            (*wallet3.wallet.address().hash(), 3),
-        ];
+        let identity_a = AirdropIdentity::Address(wallet1.wallet.address().into());
+        let identity_b = AirdropIdentity::Address(wallet2.wallet.address().into());
+        let identity_c = AirdropIdentity::Address(wallet3.wallet.address().into());
+        let minter = TokenIdentity::ContractId(deploy_wallet.contract_id);
         let key = 0;
         let num_leaves = 3;
+        let airdrop_leaves: [(AirdropIdentity, u64); 3] = [
```

Comment:
It might be useful to add a helper function to get the airdrop leaves and identities as this is repeated across tests.